frontend/src/app/main-page/main-page.component.ts

```diff
@@ -1,15 +1,12 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-main-page',
   templateUrl: './main-page.component.html',
   styleUrls: ['./main-page.component.scss']
 })
-export class MainPageComponent implements OnInit {
+export class MainPageComponent  {
 
   constructor() { }
```

Comment:
I think we can remove constructor method, because we don't do in it anything.

Reply:
done